src/components/Currency/hooks.ts

```diff
@@ -39,7 +39,14 @@ function useHandlers({
         event.stopPropagation();
 
         const { value: nextValue } = event.target;
-        updateValuesHandler({ value: nextValue ? parseFloat(nextValue) : undefined, currency: currency.label });
+        const isNumber = /^\d*[\.\,]?\d*$/;
+
+        if (isNumber.test(nextValue)) {
+            updateValuesHandler({
+                value: nextValue ? nextValue : '',
```

Comment:
Your linter should have complaint here. We can simplify it as:

```suggestion
                value: nextValue || '',
```

Reply:
done ;)